README.md

```diff
@@ -51,6 +51,8 @@ The following workflow is for a new **Helm** operator:
 
 ### Install the Operator SDK CLI
 
+Ensure installation of specific version of Operator SDK CLI for the same version of the Quick Start that is being followed.
```

Comment:
I would add a note like this after line 56:

```
 **Note:** Make sure to follow the release versioned documentation if you are installing from a release. e.g.: For version 0.8.1 see docs in https://github.com/operator-framework/operator-sdk/tree/v0.8.1. 
```

cc @joelanford WDYT ^

Reply:
This is a good idea. Here's my wordsmithed version:

> **Note:** This guide is for the current master branch. If you are using a release version of the SDK, make sure to follow the documentation for that version. e.g.: For version 0.8.1, see docs in https://github.com/operator-framework/operator-sdk/tree/v0.8.1.

I'd say remove the "Ensure installation of specfic version..." line and add the note after the "Follow the steps in the installation guide..." line.

Reply:
@Jmainguy actually I would strike `This guide is for the current master branch`, since we would have to constantly change that back and forth when we cut releases.

> **Note:** If you are using a release version of the SDK, make sure to follow the documentation for that version. e.g.: For version 0.8.1, see docs in https://github.com/operator-framework/operator-sdk/tree/v0.8.1.

Reply:
@LiliC @joelanford good suggestions and very good wordsmithing. Updated PR to reflect Joe's latest version, give it a look when yall get a chance.